### PR TITLE
Moved controllers to account folder 

### PR DESCRIPTION
**Pull request by shivendra-webkul**
## Issue Reference
Moved  account controllers to account folder

### CODE REVIEW

File: packages/Webkul/Shop/src/Http/Resources/WishlistResource.php [hunk 1]
```diff
@@ -15,8 +15,9 @@ class WishlistResource extends JsonResource
     public function toArray($request)
     {
         return [
-            'id'   => $this->id,
-            'item' => new ProductResource($this->product),
+            'id'    => $this->id,
+            'color' => $this->product->color,
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Http/Resources/WishlistResource.php:19]: Move this to product resource.

File: packages/Webkul/Shop/src/Resources/views/customers/account/wishlist/index.blade.php [hunk 2]
```diff
@@ -148,6 +148,10 @@ class=""
                                 product_id: id,
                             })
                             .then(response => {
+                                if (response.data.redirect) {
+                                    window.location.replace(response.data.data);
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Resources/views/customers/account/wishlist/index.blade.php:152]: window.location.href

File: packages/Webkul/Shop/src/Http/Resources/WishlistResource.php (hunk 1 above)
Comment (shivendra-webkul) [reply packages/Webkul/Shop/src/Http/Resources/WishlistResource.php]: Done

File: packages/Webkul/Shop/src/Http/Controllers/API/WishlistController.php [hunk 3]
```diff
@@ -89,50 +89,48 @@ public function store(): JsonResource
...
             }
+            
+            return new JsonResource([
+                'redirect' => true,
+                'data'     => route('shop.productOrCategory.index', $wishlistItem->product->url_key),
+                'message'  => trans('shop::app.checkout.cart.integrity.missing_options'),
+            ]);
+            
         } catch (\Exception $exception) {
+            report($exception);
```
[the remark this answers is not in the document]
Comment (shivendra-webkul) [reply packages/Webkul/Shop/src/Http/Controllers/API/WishlistController.php]: Done

File: packages/Webkul/Shop/src/Resources/views/customers/account/wishlist/index.blade.php (hunk 2 above)
Comment (shivendra-webkul) [reply packages/Webkul/Shop/src/Resources/views/customers/account/wishlist/index.blade.php]: done

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**